src/components/navbar/NavbarDropdown.vue

```diff
@@ -89,3 +89,8 @@ export default {
     }
 }
 </script>
+<style lang="scss" scoped>
+.is-boxed {
+    display: none;
+}
+</style>
```

Comment:
Thanks for this PR. I will need to make some tests.
Can you please add this code in `scss/components/_navbar.scss` instead? Buefy do not use scoped styles.